portfolio/src/main/webapp/script.js

```diff
@@ -39,4 +39,20 @@ function addRandomSoup() {
     //Add it to the page. 
     const soupContainer = document.getElementById('soup-container');
     soupContainer.innerText = soup;
+}
+
+function addRandom(lsts) {
+    const lst = lsts[Math.floor(Math.random() * lsts.length)];
+    return lst
```

Comment:
nit: Can we return the result directly? Up to you!

```suggestion
    return lsts[Math.floor(Math.random() * lsts.length)];
```